skins/laika/src/components/pages/donation_confirmation/MembershipInfo.vue

```diff
@@ -3,9 +3,19 @@
         <div class="column is-half">
             <h2 class="title is-size-5">{{ $t( 'donation_confirmation_membership_call_to_action_title' ) }}</h2>
             <p>{{ $t( 'donation_confirmation_membership_call_to_action_text' ) }}</p>
-            <p><b-button type="is-primary is-main">{{ $t('donation_confirmation_membership_button') }}</b-button></p>
+            <p>
+                <a href="#">
```

Comment:
@tzhelyazkova This link actually needs to be like on cat17 where it leads to the membership application page with the form filled out but I did not get around doing that.

See cat17 for reference: `skins/cat17/templates/partials/donation_confirmation/membership_advantages.html.twig`, line 17.

Reply:
You can already generate the link target with the available data on the confirmation page, do you want to change the link in a different PR or can you squeeze that in?